examples/format-imperative/.expected/diff.patch

```diff
@@ -1,37 +1,27 @@
 diff --git a/resources.yaml b/resources.yaml
-index 968a0f4..a33d07a 100644
+index f95f912..d9b7230 100644
 --- a/resources.yaml
 +++ b/resources.yaml
-@@ -1,22 +1,22 @@
- apiVersion: apps/v1
+@@ -2,13 +2,13 @@ apiVersion: apps/v1
  kind: Deployment
  metadata:
--  labels:
--    foo: bar
--    env: dev
+   labels:
+-     foo: bar
+-     env: dev
++    foo: bar
++    env: dev
    name: my-nginx
    namespace: my-space
-+  labels:
-+    env: dev
-+    foo: bar
  spec:
-+  replicas: 3
    selector:
--    matchLabels:
--      app: nginx
-     spec:
-       containers:
--      - image: nginx:1.14.2
--        name: nginx
--        ports:
--        - containerPort: 80
--  replicas: 3
-+        - name: nginx
-+          image: nginx:1.14.2
-+          ports:
-+            - containerPort: 80
+-   matchLabels:
 +    matchLabels:
-+      app: nginx
+       app: nginx
+   replicas: 3
    template:
-     metadata:
-       labels:
+@@ -20,4 +20,4 @@ spec:
+       - name: nginx
+         image: nginx:1.14.2
+         ports:
+-          - containerPort: 80
++        - containerPort: 80
```

Comment:
@phanimarupaka It seems `format` doesn't change the order of the fields. But it can still enforce consistent indentation.

Reply:
It doesn't. Functions in general doesn't change the indentation. Orchestrator does. In this case, kpt is just preserving the indentation of input resources to minimize diff.

https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/examples/format-imperative/resources.yaml#L18